FlashCardView.py

```diff
@@ -38,12 +34,16 @@
 from Globales import guardar
 from Globales import Dialog
 
+import gi
+from gi.repository import Gtk, GObject, Pango
+gi.require_version('Gtk', '3.0')
+
```

Comment:
May cause PyGObject run-time warnings if any of the imports above this point import Gtk.  You've probably done this because of "_E402 module level import not at top of file_", but the error is unavoidable.  Add a `.flake8` file instead, see Letters activity commits for example.

On the other hand, the `gi.require_version` calls have been religiously added, prior to your work, to almost every source file.  Only the first source file used in each system process needs the call added.